common/src/main/java/com/hibiscusmc/hmccosmetics/gui/Menus.java

```diff
@@ -39,6 +39,8 @@ public static boolean hasMenu(Menu menu) {
         return MENUS.containsValue(menu);
     }
 
+    public static Menu getDefaultMenu() { return MENUS.get(Settings.getDefaultMenu()); }
```

Comment:
Make this use the getMenu method, as all menu names are uppercase internally and the default config can have either lowercase or uppercase right now.